pydis_core/utils/paste_service.py

```diff
@@ -0,0 +1,95 @@
+from aiohttp import ClientConnectorError, ClientSession
+
+from pydis_core.utils import logging
+
+log = logging.get_logger(__name__)
+
+FAILED_REQUEST_ATTEMPTS = 3
+MAX_PASTE_SIZE = 256 * 1024  # 256kB
+
+
+class PasteUploadError(Exception):
+    """Raised when an error is encountered uploading to the paste service."""
+
+
+class PasteTooLongError(Exception):
+    """Raised when content is too large to upload to the paste service."""
+
+
+async def send_to_paste_service(
+    *,
+    file_name: str,
```

Comment:
The name field is optional in the API, so I don't think we need to require it.

Reply:
Updated in the [force push](https://github.com/python-discord/bot-core/compare/506329a56f2642af5ff7eed53a5ced1838fe4336..801e1e5151cde2d01024fc025b8b688558720070)